src/modules/file-share/file-share-list/file-share-list.js

```diff
@@ -0,0 +1,53 @@
+(function() {
+    "use strict";
+
+    var app = angular.module("TendrlModule");
+
+    app.controller("fileShareController", fileShareController);
+
+    /*@ngInject*/
+    function fileShareController($scope, $interval, utils, config) {
+        var vm = this;
+
+        vm.setupFileShareListData = setupFileShareListData;
+        init();
+
+        function init() {
+            var list = utils.getVolumeDetails();
+            vm.fileShareList = vm.setupFileShareListData(list);
+        }
+
+        function setupFileShareListData(list) {
+            var fileShareList=[], fileShare = {}, i, len = list.length;
+            for ( i = 0; i < len; i++ ) {
+                var volume = list[i];
```

Comment:
And use either file share or volume in the controller as both means the same only I guess. Its better to use file share as the same name is there in UX design.